omnibus/package-scripts/agent/posttrans

```diff
@@ -41,10 +41,10 @@ if command -v systemctl >/dev/null 2>&1; then
 elif command -v initctl >/dev/null 2>&1; then
     # start/stop policy is already defined in the upstart job file
     :
-elif command -v update-rc.d >/dev/null 2>&1; then
```

Comment:
Shouldn't we keep this `elif` block after the new `elif` block you added? Can we know it's never used?

Reply:
I added it specifically for SUSE 11 (this block didn't exist before). Since I realised it doesn't work on SUSE 11 either by default, I removed it. We could keep it within the `$SUSE_SYSVINIT_SUPPORT` elif block (in case someone does use `update-rc.d` on SUSE 11), in which case I'll have to re-add it back to the prerm script as well.